src/main/java/com/management/HumanResources/model/Feedback.java

```diff
@@ -0,0 +1,17 @@
+package com.management.HumanResources.model;
+
+import org.json.JSONObject;
+
+public class Feedback {
+    private int id;
+    private String type;
+    private String description;
+
+    public String feedbackData() {
```

Comment:
I appreciate you making this method. I used this technique in the `EmployeeTime` model just to hide extraneous values from being entered into the database. Fortunately there isn't much business logic to go along with submitting and feedback so you don't really need to convert it to Json here. You can just send the plain `feedback` object itself in your getFeedback method and Webclient will serialize to Json by default.